deploy/sre-build-test/70-stuck-builds.PrometheusRule.yaml

```diff
@@ -11,7 +11,12 @@ spec:
   - name: sre-alerts-stuck-builds
     rules:
     - alert: StuckNewBuilds3MinSRE
-      expr: count(openshift_build_active_time_seconds{phase="New"} < time() - 180) > 0
+      expr: count(openshift_build_active_time_seconds{phase="New",exported_namespace=~"^openshift-build-test.*"} < time() - 180) > 0
+      labels:
+        severity: warning
+        namespace: openshift-monitoring
+    - alert: StuckPendingBuilds3MinSRE
+      expr: count(openshift_build_active_time_seconds{phase="Pending",exported_namespace=~"^openshift-build-test.*"} < time() - 180) > 0
```

Comment:
Is it possible to write these using `for` instead of the time calculation?

(I'll take the blame for using a time calculation in the first place, in other rules. 😅 )

Reply:
haha, thanks for taking the blame ;)

Didn't look at changing this one other than adding the Pending variant.

Reply:
To be clear, if we want to change the rules I recommend tracking elsewhere, I didn't want to get into changing alerts except to point at limited namespaces.  I noticed the "Pending" state so added that alert..

Reply:
No, I agree. I had thought a "clean as you go" approach would work, but a wholistic approach is probably better.